### PR TITLE
zh-translation: docs/root/install/ref_configs.rst

### PR DESCRIPTION
**Pull request by vgbhfive**
For an explanation of how to fill out the fields, please see the relevant section
in [PULL_REQUESTS.md](https://github.com/envoyproxy/envoy/blob/master/PULL_REQUESTS.md)

Commit Message:
Additional Description:
Risk Level:
Testing:
Docs Changes:  docs/root/install/ref_configs.rst
Release Notes:
[Optional Runtime guard:]
[Optional Fixes #Issue] #216 
[Optional Deprecated:]

### CODE REVIEW

File: docs/root/install/ref_configs.rst [hunk 1]
```diff
@@ -1,58 +1,42 @@
...
-* :ref:`Front proxy <deployment_type_front_proxy>`
-* :ref:`Double proxy <deployment_type_double_proxy>`
+* :ref:`服务到服务 <deployment_type_service_to_service>`
+* :ref:`前置代理 <deployment_type_front_proxy>`
+* :ref:`双重代理 <deployment_type_double_proxy>`
 
-The goal of this set of example configurations is to demonstrate the full capabilities of Envoy in
-a complex deployment. All features will not be applicable to all use cases. For full documentation
-see the :ref:`configuration reference <config>`.
+这套示例配置的目的是为了演示 Envoy 在复杂部署中的全部功能。所有特性并不适用于所有示例。有关完整文档内容请 :ref:`参考配置 <config>`.
```
Comment (helight) [inline docs/root/install/ref_configs.rst]:
```suggestion
这套示例配置的目的是为了演示 Envoy 在复杂部署中的全部功能。所有特性并不适用于所有示例。有关完整文档内容请看 :ref:`参考配置 <config>`.
```

File: docs/root/install/ref_configs.rst [hunk 2]
```diff
@@ -1,58 +1,42 @@
...
-  disable this or enable `Zipkin <https://zipkin.io>`_ or `Datadog <https://datadoghq.com>`_ tracing, delete or
-  change the :ref:`tracing configuration <envoy_api_file_envoy/config/trace/v2/trace.proto>` accordingly.
-* The configuration demonstrates the use of a :ref:`global rate limiting service
-  <arch_overview_global_rate_limit>`. To disable this delete the :ref:`rate limit configuration
-  <config_rate_limit_service>`.
-* :ref:`Route discovery service <config_http_conn_man_rds>` is configured for the service to service
-  reference configuration and it is assumed to be running at `rds.yourcompany.net`.
-* :ref:`Cluster discovery service <config_cluster_manager_cds>` is configured for the service to
-  service reference configuration and it is assumed that be running at `cds.yourcompany.net`.
+先前的命令会使用在 `configgen.py` 内部定义的一些变量生成三种完全扩展的配置。关于不同的扩展是如何工作的详细信息，请参考 `configgen.py` 的内部注释。
```
Comment (helight) [inline docs/root/install/ref_configs.rst]:
```suggestion
前面的命令会使用在 `configgen.py` 内部定义的一些变量生成三种完全扩展的配置。关于不同的扩展是如何工作的详细信息，请参考 `configgen.py` 的内部注释。
```

File: docs/root/install/ref_configs.rst [hunk 3]
```diff
@@ -1,58 +1,42 @@
...
+
+关于示例配置的一些注意事项：
+
+* 假设一个 :ref:`端点发现服务（EDS） <arch_overview_service_discovery_types_eds>` 的实例运行在 `discovery.yourcompany.net` 上。
+* 假设为 `yourcompany.net` 设置了 DNS。在配置模板中查找不同的实例。
+* 为 `LightStep <https://lightstep.com/>`_ 配置了跟踪。要禁用此功能请开启 `Zipkin <https://zipkin.io>`_ 或者 `Datadog <https://datadoghq.com>`_ 跟踪，
```
Comment (helight) [inline docs/root/install/ref_configs.rst:38]:
```suggestion
* 为 `LightStep <https://lightstep.com/>`_ 配置了跟踪。要禁用或者开启 `Zipkin <https://zipkin.io>`_ 或者 `Datadog <https://datadoghq.com>`_ 跟踪，
```

**Review by helight — COMMENTED** [review]
一点建议，整体非常好，赞

File: docs/root/install/ref_configs.rst (hunk 3 above)
Comment (vgbhfive) [reply docs/root/install/ref_configs.rst:38]: 这里的翻译我觉得是没有问题的。

**Comment by vgbhfive** [comment]
> 一点建议，整体非常好，赞

指正的地方，我已经修复了，感谢。

**Comment by helight** [comment]
LGTM
/approve

**Pull request merged**